src/commands/stepfunctions/__tests__/helpers.test.ts

```diff
@@ -288,71 +287,61 @@ describe('stepfunctions command helpers tests', () => {
         },
         End: true,
       }
-      expect(
-        shouldUpdateStepForStepFunctionContextInjection(step, context, 'Step Functions StartExecution')
-      ).toBeTruthy()
+
+      const changed = injectContextForStepFunctions(step, context, 'Step Functions StartExecution')
+      expect(changed).toBeTruthy()
+      expect(step.Parameters?.Input).toEqual({'CONTEXT.$': 'States.JsonMerge($$, $, false)'})
     })
 
-    test('is true for undefined', () => {
+    test('is true for an empty object', () => {
```

Comment:
GitHub doesn't show the changes well. New lines 296–345 were modified from old lines 281–339